plugins/Goals/TranslationHelper.php

```diff
@@ -14,6 +14,33 @@
 
 class TranslationHelper
 {
+
+    public function getTranslationForGoalMetricCategoryText($type, $category)
+    {
+        // Type is either 'Goals' or 'Ecommerce'
+        switch ($type) {
+            case 'Goals':
+                $subKey = 'Goals';
+                break;
+
+            case 'Ecommerce':
+                $subKey = 'Sales';
+                break;
+
+            default:
+                return '';
+        }
```

Comment:
Would it make sense to have 2 methods instead? E.g. `translateGoal…` and `translateEcommerce…`